src/Stripe.net/Services/_base/AnyOf.cs

```diff
@@ -21,6 +21,85 @@ public abstract class AnyOf : IAnyOf
         public override string ToString() => this.Value == null ? "AnyOf(null)" : this.Value.ToString();
     }
 
+    /// <summary>
+    /// <see cref="AnyOf{T}"/> is a generic class that can hold a value of one type.
```

Comment:
It might make this type a bit more readable in the context of the file if we kept this as T1, with the idea being that we set the expectation that for any AnyOf, the first type is T1, etc.  Not a blocking thing, though.

Reply:
That's fair. I think in the context of the internals of this class, `T` makes more sense because there'll never be a `T2` to account for? But I don't feel super strongly either way.

Reply:
Same; I think your rationale makes sense.  The only exception is _maybe_ the case where I'm wading thru several variants of AnyOf and consistency here might make that easier, but I suspect the fact that this specific case hasn't come up yet means this doesn't/won't get touched often.